utils/vibe/internal/meta/codegen.d

```diff
@@ -78,7 +78,7 @@ template getSymbols(T)
 			alias Implementation = TypeTuple!();
 	}
 
-	alias getSymbols = NoDuplicates!(Implementation!T);
+	alias getSymbols = NoDuplicates!(staticMap!(Unqual, Implementation!T));
```

Comment:
This has compile-time performance implications.

Reply:
Isn't this a breaking change? Wouldn't the original say `AliasSeq!(int, const(int))` for `int[const(int)]`? alias parameters seem to allow `const(int)` to go through, whereas `Unqual!(const(int))` turns into `int`

Reply:
@schveiguy Oops you're right

Reply:
Actually I think basic types are ignored anyway

Reply:
Yep, you are right, I didn't realize that.

Reply:
Instead of this can you apply Unqual at the leaf instantiations?  That would prevent the need for static map.